### PR TITLE
test glue/replicate.js and add nyc for coverage

### PR DESCRIPTION
**Pull request by staltz**

### CODE REVIEW

File: test/glue.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+const tape = require('tape')
+const ssbKeys = require('ssb-keys')
+const Server = require('scuttle-testbot')
+const u = require('./util')
+
+const followedKeys = ssbKeys.generate()
+const blockedKeys = ssbKeys.generate()
+
+tape('listen to hopStream and requests replication of follows', function (t) {
```
Comment (staltz) [inline test/glue.js:9]: Mocks the replicate plugin to check that `sbot.replicate.request` is called when the glue/replicate file is activated, when a follow occurs.

File: test/glue.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      }
+    })
+
+  sbot.friends.follow(followedKeys.id, {}, (err, msg) => {
+    t.error(err, 'no error when following')
+    t.ok(msg, 'contact msg is truthy')
+  })
+})
+
+tape('listen to hopStream and stops replication of blocks', function (t) {
```
Comment (staltz) [inline test/glue.js:42]: Mocks the replicate plugin to check that `sbot.replicate.block` is called, both when blocking or unblocking.

**Review by arj03 — APPROVED** [review]
Looks good, out of curiosity what are the nyc numbers for this module?

**Comment by staltz** [comment]
@arj03 

## Before

Screenshot from 2021-06-03 17-02-29

## After

Screenshot from 2021-06-03 17-02-46

**Pull request merged**